lib/gui/components/update-notifier/templates/update-notifier-modal.tpl.html

```diff
@@ -6,6 +6,9 @@ <h4 class="modal-title">New Update Available!</h4>
 <div class="modal-body">
   <div class="modal-text">
     <p>A new version of Etcher is available for download</p>
+    <a os-open-external="https://github.com/resin-io/etcher/blob/master/CHANGELOG.md#readme">
```

Comment:
Why the `#readme` hash? Should it be something like https://github.com/resin-io/etcher/blob/master/CHANGELOG.md#change-log?